tests/generator/tests/nativeenum.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using Foundation;
+using ObjCRuntime;
+
+namespace NS {
+	[Native]
+	public enum MyEnum6 : long {
```

Comment:
It looks weird to start at "6", but I have more local tests that will fill in the missing numbering 😄